bigquery_etl/metadata/parse_metadata.py

```diff
@@ -256,10 +261,9 @@ def from_file(cls, metadata_file):
                 if "owners" in metadata:
                     owners = metadata["owners"]
                     for i, owner in enumerate(metadata["owners"]):
-                        label = owner.split("@")[0]
-                        if not Metadata.is_valid_label(label):
-                            label = ""
-                        labels[f"owner{i+1}"] = label
+                        if not is_email(owner):
```

Comment:
I think emails cannot be labels because "@" is not allowed?
https://cloud.google.com/bigquery/docs/labels-intro#requirements

Reply:
Ahh fair enough, I'll update this to just skip non-email labels then